### PR TITLE
OSDOCS-12192: table of parameter explainers for config.yaml

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.17, 4.18

Issue:
[OSDOCS-12192](https://issues.redhat.com/browse/OSDOCS-12192)

Link to docs preview:
[MicroShift configuration parameters table](https://83051--ocpdocs-pr.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-yaml.html#microshift-config-parameters-table_microshift-configuring)
[Revised flow of the MicroShift config section](https://83051--ocpdocs-pr.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-yaml.html#microshift-config-yaml_microshift-configuring)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by DanielFroehlich — APPROVED** [review]
/LGTM

File: modules/microshift-config-parameters-table.adoc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+|`Managed` or `Removed`
+|Default router status.
+
+|`kubelet`
+|See the {microshift-short} low-latency instructions
+|Parameter for passthrough configuration of the kubelet node agent. Used for low-latency configuration. Default value is null.
+
+|`manifests`
+|`list of paths`
+|The locations on the file system to scan for `kustomization` files to use to load manifests. Set to a list of paths to scan only those paths. Set to an empty list to disable loading manifests. The entries in the list can be glob patterns to match multiple subdirectories. Default values are `/usr/lib/microshift/manifests`, `/usr/lib/microshift/manifests.d/`, `/etc/microshift/manifests`, and `/etc/microshift/manifests.d/`
```
Comment (adellape) [inline modules/microshift-config-parameters-table.adoc:95]: Punctuation:
```suggestion
|The locations on the file system to scan for `kustomization` files to use to load manifests. Set to a list of paths to scan only those paths. Set to an empty list to disable loading manifests. The entries in the list can be glob patterns to match multiple subdirectories. Default values are `/usr/lib/microshift/manifests`, `/usr/lib/microshift/manifests.d/`, `/etc/microshift/manifests`, and `/etc/microshift/manifests.d/`.
```

File: modules/microshift-config-parameters-table.adoc [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+|`list`
+|Optional. Add a list of explicit DNS names. Leading wildcards are allowed. If no names are provided, the implicit names are extracted from the certificates.
+
+|`subjectAltNames`
+|Fully qualified domain names (FQDNs), wildcards such as `*.domain.com`, or IP addresses
+|Subject Alternative Names for API server certificates. SANs indicate all of the domain names and IP addresses that are secured by a certificate.
+
+|`debugging.logLevel`
+|`Normal`, `Debug`, `Trace`, or `TraceAll`
+|Log verbosity.
```
Comment (dhensel-rh) [inline modules/microshift-config-parameters-table.adoc:58]: Log verbosity. Default is normal
```suggestion
|Log verbosity.
```

File: modules/microshift-config-parameters-table.adoc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+|IP address block
+|A block of virtual IP addresses for Kubernetes services. IP address pool for services. IPv4 is the default. Dual-stack entries are supported. The first entry in this field is immutable after {microshift-short} starts. Default range is `10.43.0.0/16`.
+
+|`network.serviceNodePortRange`
+|`range`
+|The port range allowed for Kubernetes services of type `NodePort`. If not specified, the default range of 30000-32767 is used. Services without a `NodePort` specified are automatically allocated one from this range. This parameter can be updated after {microshift-short} starts.
+
+|`node.hostnameOverride`
+|`string`
+|The name of the node. The default value is the hostname. If non-empty, this string is used to identify the node instead of the hostname.
```
Comment (dhensel-rh) [inline modules/microshift-config-parameters-table.adoc:111]: I think we need the immutable verbage here.

File: modules/microshift-config-parameters-table.adoc [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+The following table explains {microshift-short} configuration YAML parameters and valid values for each:
+
+.{microshift-short} `config.yaml` parameters
+[cols="3","20%,10%,70%",options="header"]
+|===
+|Field|Type|Description
+
+|`advertiseAddress`
+|`array`
```
Comment (dhensel-rh) [inline modules/microshift-config-parameters-table.adoc:17]: I'm not sure why, but I'm troubled by the use of  `array` here, and other instances below.

Comment (ShaunaDiaz) [reply modules/microshift-config-parameters-table.adoc:17]: Standard for docs. Is it the backticks that trouble you, or would you prefer another word here?

**Pull request merged**